broadlink/cover.py

```diff
@@ -1,5 +1,6 @@
 """Support for covers."""
 import time
+import struct
```

Comment:
Remove unused import.

Reply:
Thanks for your constructive comments. All necessary changes have been made.